superset-frontend/src/utils/getClientErrorObject.ts

```diff
@@ -48,6 +50,12 @@ export default function getClientErrorObject(
           .json()
           .then(errorJson => {
             let error = { ...responseObject, ...errorJson };
+
+            // Backwards compatibility for old error renderers with the new error object
+            if (error.errors && error.errors.length > 0) {
```

Comment:
I suspect this new line cause some error_details logged as empty.
this is how front-end use error field:
https://github.com/apache/incubator-superset/blob/38a6bd79da2be8c13a89a5f67f77faeb55c4e08b/superset-frontend/src/chart/chartAction.js#L403